src/CommonGLPI.php

```diff
@@ -686,12 +686,37 @@ public static function displayStandardTab(CommonGLPI $item, $tab, $withtemplate
                     return $ret;
                 }
 
+                // Check if we're creating an object from template and this is not the main tab
+                if ($withtemplate == 2 && $tabnum != 'main' && $item instanceof CommonDBTM) {
+                    // Display generic message for template creation
+                    $template = <<<HTML
+                    <div class="alert alert-info d-flex">
+                        <div class="me-2">
+                            <i class="ti ti-info-circle"></i>
+                        </div>
+                        <div>
+                            <strong>%s</strong><br>
+                            %s
+                        </div>
+                    </div>
+                    HTML;
+                    echo sprintf(
+                        $template,
+                        __s('Creating from template'),
+                        __s('You are currently creating an object from a template. You need to save it, in the main tab, before editing data in other tabs.')
+                    );
+                    return true;
```

Comment:
Right now the tab content is not displayed with this change:

<img width="1333" height="513" alt="image" src="https://github.com/user-attachments/assets/c4d4640b-61e5-48dd-83af-df1f2f9692bc" />

Is this ok for you @orthagh?

I am not sure because the user lose the ability to preview what will be created.

Reply:
No, we don't want a return indeed. We must force read-only right if possible.